### ISSUE TITLE
`terminal.integrated.enablePersistentSessions` doesn't work.

### ISSUE THREAD

**Issue by byehack** [microsoft/vscode#129395]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.58.2
- OS Version: win10 20h2

Steps to Reproduce:

https://user-images.githubusercontent.com/26390254/126983513-374a01c8-613a-4a26-8f29-be56b1f26d1c.mp4

**Comment by meganrogge** [comment]
I can't reproduce the issue. Are you seeing any errors in dev tools?

![recording (39)](https://user-images.githubusercontent.com/29464607/127013997-1d07735a-548e-4536-8c79-65c20e72c5bd.gif)

**Comment by byehack** [comment]
![image](https://user-images.githubusercontent.com/26390254/127022924-ed4e6ab2-40bb-4ece-8545-ab350bd9577a.png)


i can see only this error on each reload.

**Comment by meganrogge** [comment]
Can you reproduce this if you comment out your settings?

**Comment by byehack** [comment]
> Can you reproduce this if you comment out your settings?

yes i can reproduce this with all settings commented out and all extensions disabled.

**Comment by meganrogge** [comment]
Can you reproduce this in insider's?

**Comment by byehack** [comment]
yes even insider's.

**Comment by meganrogge** [comment]
@Tyriar thoughts?

**Comment by Tyriar** [comment]
Did you create the terminals when the setting was unchecked and reload when it was checked? The terminals keep whatever mode was set when they were created.

**Comment by byehack** [comment]
@Tyriar 

https://user-images.githubusercontent.com/26390254/127160649-b6af4cb4-bc3b-45a2-9eca-5521db6595f5.mp4

**Comment by urielvan** [comment]
I've met same problem here. As you can see, it works fine if I move the terminal into the editor area. But the terminal disappears if I move it back to panel area.

Btw, my project listen to port 8000 by default, but port 8000 seems to be used even if the terminal seems to be missing. I have to close and restart vscode again if I want to use port 8000.

![terminal](https://user-images.githubusercontent.com/14124339/133871284-69d50e2a-b413-4f8c-8cd5-2362589bf2ad.gif)
